lib/record_store/record/txt.rb

```diff
@@ -11,13 +11,17 @@ def initialize(record)
     end
 
     def to_s
-      "[TXTRecord] #{fqdn} #{ttl} IN TXT \"#{txtdata}\""
+      "[TXTRecord] #{fqdn} #{ttl} IN TXT \"#{rdata_txt}\""
```

Comment:
Maybe missed this one too? I might be wrong though. I notice you have the escape sequence characters in the ones you haven't deleted.

Reply:
So I left this because of the quotes I put around TXT record contents. I'm open to remove them if there's consensus around doing that.

Reply:
Going to approve, but would be dope to get rid of these methods! 

I don't think the quotation marks add enough value for you to make an exception to the pattern you've created :)

Reply:
Will 🔥 in a follow-up PR 👌